.github/workflows/helm-chart-repo.yaml

```diff
@@ -93,7 +98,7 @@ jobs:
       - name: Create Pull Request
         uses: devops-infra/action-pull-request@v0.4.2
         with:
-          github_token: ${{ secrets.GH_PAT }}
+          github_token: ${{ secrets.GH_PAT }}  # TODO: Do we remember why we use GH_PAT here and not GITHUB_TOKEN?
```

Comment:
Was it because we didn't want to give `write` perms to `GITHUB_TOKEN`?

Reply:
It's because we _can't_.

We earlier found that an action running with `GITHUB_TOKEN` _**cannot**_ create a PR, though we tried all sorts of things.

Root cause seems to be a combination of 
(a) The same permission that allows creation of PRs also allows approval, and automatic approval of PRs from uncontrolled public forks is hazardous
(b) Policies are tighter within the `Azure` organization

Reply:
So we've been using PATs since then as a workaround. 

But it seems there's a third option we should investigate:

It seems we can set up an App to generate a token for us [[Link](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens)]..